src/layer/wms.ts

```diff
@@ -89,16 +89,16 @@ export function wmsGetMapUrl(
   }
 
   if (!params.fromTime) {
-    queryParams.time = moment.utc(params.toTime).format('YYYY-MM-DDTHH:mm:ss') + 'Z';
+    queryParams.time = moment.utc(params.toTime).format('YYYY-MM-DD');
```

Comment:
why was this changed to date only?

Reply:
Proba-V doesn't support time in single date mode. Setting this universally makes sense as we're actually searching for date and not datetime in this case. Other providers support this as well.

Reply:
I think this should be handled by client (eo-browser in this case) and not by library. WMS handler should be agnostic and  doesn't know anything about proba-v or any other wms service. Just because eo-browser uses some wms service which doesn't support time,  that doesn't mean that 
- all wms services supports date only
- no one will ever want to set time, after all parameter is called `time` and not date

Personally, I would do this in eob

Reply:
I generally agree. But am also not sure EOB is the right place to handle these peculiarities. All in all the right way of addressing these would be to read getCapabilities more rigorously and then act accordingly (not sure if it does indeed contain all the required info). But am not sure if this is something worth pursuing at this point. 

Perhaps I would rather suggest creating a separate layer for the WMT MS and handle it there. What do you think?

Reply:
Yes, I think this could work.